lib/sqlalchemy/sql/traversals.py

```diff
@@ -211,8 +200,8 @@ class GenerativeOnTraversal(HasShallowCopy):
     __slots__ = ()
 
     def _generate(
-        self: SelfGenerativeOnTraversal,
-    ) -> SelfGenerativeOnTraversal:
+        self,
```

Comment:
can you remove the tailing comma, so that black formats it inline?

Reply:
Fixed, thanks for poiting on it